entries/t/trustage.com.json

```
@@ -0,0 +1,12 @@
+{
+    "Trustage": {
+      "domain": "trustage.com",
+      "url": "https://www.trustage.com/",
+      "tfa": [
+        "sms"
+      ],
```

```suggestion
      "tfa": [
        "call",
        "sms",
        "email"
      ],
```

Their [help page](https://www.trustage.com/my-account/help/online-account-information-faqs) ("How do I sign in with 2-Step Security?" and "What options do I have with 2-Step Security?") says they support sending via "phone, text or email".

---

@Carlgo11